jelares/portfolio/src/main/webapp/css/homeStyle.css

```diff
@@ -34,7 +43,7 @@ h1, h2, h3, h4 {
 
     font-family: 'Roboto', sans-serif;
     font-weight: 400;
-    font-size: 4em;
+    font-size: 4rem;
```

Comment:
Typo?

Reply:
No, I tried to use rem for all the styling because that's the bootstrap standard